apps/web/pages/data.tsx

```diff
@@ -148,24 +77,16 @@ export default function DataExplorerScreen() {
               }}>
               Execute SQL
             </button>
-            <button
-              className="btn"
-              onClick={async () => {
-                // @ts-expect-error
-                const res = await trpcClient.mutation('createDbUser', {})
```

Comment:
@tonyxiao Why was this needed (`create db user`)? Can it be put into a debug menu?

Reply:
yes, it's debug only. I need to implement the logic of creating DB user on demand (e.g. when you access the database url) and also probably via postgres trigger (on auth.users creation)